docs/troubleshooting/Linux/index.md

```diff
@@ -0,0 +1,25 @@
+```{eval-rst}
+.. image:: ../../images/cropped-plus3it-logo-cmyk.png
+   :width: 140px
+   :alt: Powered by Plus3 IT Systems
+   :align: right
+   :target: https://www.plus3it.com
+```
+<br>
+
+# Linux Log-Files
+
+The logfiles to pay most attention to when running Watchmaker on Enterprise Linux distros (Red Hat, CentOS, Oracle Enterprise, etc.) are as follows:
+
+```{toctree}
+:maxdepth: 1
+salt_call.debug.log.md
+watchmaker.log.md
```

Comment:
I'd suggest putting the watchmaker.log file first, since it's the one that will actually contain an exception on execution failure. And that exception typically ought to point at the root cause. The salt log is only useful when the failure is a salt formula, and even then, the watchmaker exception will point to _which_ salt formula. As far as a first place to look, the watchmaker log is almost always a better place to start.

```suggestion
watchmaker.log.md
salt_call.debug.log.md
```